README.md

```diff
@@ -6,149 +6,96 @@
 
-**Getting the right openshift-ansible release**
+#### GCP Credentials
 
-Follow this release pattern and you can't go wrong:
+Place your service account key with the name `gce.json` in the root of OpenShift-Ansible.
 
-| Origin/OCP    | OpenShift-Ansible version | openshift-ansible branch |
-| ------------- | ----------------- |----------------------------------|
-| 1.3 / 3.3          | 3.3               | release-1.3 |
-| 1.4 / 3.4          | 3.4               | release-1.4 |
-| 1.5 / 3.5          | 3.5               | release-1.5 |
-| 3.*X*         | 3.*X*             | release-3.x |
+OpenShift developers can find the service account key in the [shared secrets repo](https://github.com/openshift/shared-secrets/blob/master/gce/aos-serviceaccount.json).
 
-If you're running from the openshift-ansible **master branch** we can
-only guarantee compatibility with the newest origin releases **in
-development**. Use a branch corresponding to your origin version if
-you are not running a stable release.
 
+### files_dir (do this once) TODO:REMOVE
 
-## Setup
 
-Install base dependencies:
+All files (bootstrap.ign, gce.json) should be saved to the root of this repo. Tell the playbooks where to find them:
 
-Requirements:
+```
+# Execute this from the root dir
+echo files_dir: $PWD > inventory/dynamic/gcp/group_vars/all/00_default_files_dir.yml
+```
 
-- Ansible >= 2.6.5, Ansible 2.7 is not yet supported and known to fail
-- Jinja >= 2.7
-- pyOpenSSL
-- python-lxml
+### Prepare Inventory Variables
+TODO: can we skip this step by putting these variables into the repo or is there sensitive info? ideally put them in the test directory
 
-----
+```
+wget http://haste.usersys.redhat.com/raw/uziredenic -O inventory/dynamic/gcp/group_vars/all/vars.yaml
+wget http://haste.usersys.redhat.com/raw/enahiroxoy -O inventory/dynamic/gcp/group_vars/all/vars-origin.yaml
+```
 
-Fedora:
+### Authenticate to GCP
 
 ```
-dnf install -y ansible pyOpenSSL python-cryptography python-lxml
+gcloud auth activate-service-account --quiet --key-file="gce.json"
 ```
 
-Additional requirements:
+You can confirm login with: `gcloud compute --project "openshift-gce-devel" instances list`.
+
+### Ignition Configs
 
-Logging:
+Use the [OpenShift Installer](https://github.com/openshift/installer) to generate a bootstrap ignition file that will be used by the ansible playbooks.
+
+Set [environment variables](https://github.com/openshift/installer/blob/master/docs/user/environment-variables.md) for the installer:
+```
+export OPENSHIFT_INSTALL_PLATFORM="libvirt"
+export OPENSHIFT_INSTALL_LIBVIRT_URI="qemu+tcp://192.168.122.1/system" 
+export OPENSHIFT_INSTALL_LIBVIRT_IMAGE="file:///unused" 
+export OPENSHIFT_INSTALL_CLUSTER_NAME="<yourname>" 
+export OPENSHIFT_INSTALL_BASE_DOMAIN="origin-gce.dev.openshift.com" 
+export OPENSHIFT_INSTALL_EMAIL_ADDRESS="<yourname>@redhat.com" 
+export OPENSHIFT_INSTALL_PASSWORD="none"
+export OPENSHIFT_INSTALL_PULL_SECRET_PATH="<path/to/pull_secret.json>"
```

Comment:
this is an interesting one, so the Installer docs mention that you need to go to `try.openshift.com` and download the secret however it doesn't provide much details as to what that secret is used for.

Maybe having a note here will be good to have ?

Reply:
This secret is used to a) pull quay images, b) used by telemetry to identify the cluster.

Eventually it won't be required as all images should be available and telemetry is opt-out, but at this point its still needed by many components